members/CusD80cfeebe1533156797.yaml

```diff
@@ -0,0 +1,125 @@
+    - find:
+      - selector: div.GIY1LSJGSC form
+    - fill_in:
+        - name: message
+          selector: textarea#x-auto-0-input
+          value: $MESSAGE
+          required: true
+        - name: First Name
+          selector: input#x-auto-1-input
+          value: $NAME_FIRST
+          required: false
+        - name: Last Name
+          selector: input#x-auto-2-input
+          value: $NAME_LAST
+          required: false
+    - click_on:
+        - selector: input#gwt-uid-85
+    - fill_in:
+        - name: City
+          selector: input#x-auto-5-input
+          value: $ADDRESS_CITY
+          required: false
+        - name: zip5
+          selector: input#x-auto-8-input
+          value: $ADDRESS_ZIP5
+          required: false
+        - name: email
+          selector: input#x-auto-10-input
+          value: $EMAIL
+          required: false
+    - select:
+        - name: state
+          selector: input#x-auto-6-input
+          value: $ADDRESS_STATE_FULL
+          required: false
+          options:
+            Alabama: Alabama
+            Alaska: Alaska
+            American Samoa: American Samoa
+            Arizona: Arizona
+            Arkansas: Arkansas
+            California: California
+            Colorado: Colorado
+            Connecticut: Connecticut
+            Delaware: Delaware
+            District of Columbia: District of Columbia
+            Florida: Florida
+            Georgia: Georgia
+            Guam: Guam
+            Hawaii: Hawaii
+            Idaho: Idaho
+            Illinois: Illinois
+            Indiana: Indiana
+            Iowa: Iowa
+            Kansas: Kansas
+            Kentucky: Kentucky
+            Louisiana: Louisiana
+            Maine: Maine
+            Maryland: Maryland
+            Massachusetts: Massachusetts
+            Michigan: Michigan
+            Minnesota: Minnesota
+            Mississippi: Mississippi
+            Missouri: Missouri
+            Montana: Montana
+            Nebraska: Nebraska
+            Nevada: Nevada
+            New Hampshire: New Hampshire
+            New Jersey: New Jersey
+            New Mexico: New Mexico
+            New York: New York
+            North Carolina: North Carolina
+            North Dakota: North Dakota
+            Northern Mariana Islands: Northern Mariana Islands
+            Ohio: Ohio
+            Oklahoma: Oklahoma
+            Oregon: Oregon
+            Pennsylvania: Pennsylvania
+            Puerto Rico: Puerto Rico
+            Rhode Island: Rhode Island
+            South Carolina: South Carolina
+            South Dakota: South Dakota
+            Tennessee: Tennessee
+            Texas: Texas
+            US Virgin Islands: US Virgin Islands
+            Utah: Utah
+            Vermont: Vermont
+            Virginia: Virginia
+            Washington: Washington
+            West Virginia: West Virginia
+            Wisconsin: Wisconsin
+            Wyoming: Wyoming
+        - name: Category
```

Comment:
just double checking this works as a `select`

Reply:
yeap yeap 
[![Screenshot from Gyazo](https://gyazo.com/0e7c9fb65c6b6e8fcb8a4bee694b204a/raw)](https://gyazo.com/0e7c9fb65c6b6e8fcb8a4bee694b204a)

Reply:
Thanks for checking tho @tomdionesotes 

let me know if there are more flags. If not, feel free to approve this PR. Thanks!